src/plugins/maps_legacy/public/map/map_messages.js

```diff
@@ -71,7 +71,7 @@ export const createRegionBlockedWarning = (function () {
         document.getElementsByClassName('leaflet-container'),
         (leafletDom) => {
           ReactDOM.render(
-            new RegionBlockedWarningOverlay().render(),
+            new RegionDeniedWarningOverlay().render(),
             leafletDom.appendChild(messageBlock)
           );
         }
```

Comment:
nit: We export `createRegionDeniedWarning` and `removeRegionBlockedWarning` when it was previously `createRegionBlockedWarning` and `removeRegionBlockedWarning`. So I think it should be consistent naming.

Reply:
fixed